tests/testFileEditor.js

```diff
@@ -257,14 +257,14 @@ describe('test file editor', function() {
         });
     });
 
-    describe('the example course', function() {
+    describe('the testCourse', function() {
 
         before('set up testing server', helperServer.before());
 
         after('shut down testing server', helperServer.after);
 
-        describe('disallow edits inside exampleCourse', function() {
-            badGet(badExampleCoursePathUrl);
+        describe('disallow edits inside testCourse', function() {
```

Comment:
For some reason, this is giving a status of 200 instead of 400 in the CI unit test run. Not sure why.... 

https://github.com/PrairieLearn/PrairieLearn/runs/295349014#step:9:11912

<details>
<summary>ci output</summary>

```bash
9876 passing (6m)
  1 failing

  1) test file editor
       the testCourse
         disallow edits inside testCourse
           GET to edit url with bad path
             should not load successfully:
     Error: bad status: 200
      at Request._callback (tests/testFileEditor.js:283:37)
      at Request.self.callback (node_modules/request/request.js:185:22)
      at Request.EventEmitter.emit (domain.js:476:20)
      at Request.<anonymous> (node_modules/request/request.js:1161:10)
      at Request.EventEmitter.emit (domain.js:476:20)
      at IncomingMessage.<anonymous> (node_modules/request/request.js:1083:12)
      at IncomingMessage.EventEmitter.emit (domain.js:476:20)
      at endReadableNT (_stream_readable.js:1183:12)
      at processTicksAndRejections (internal/process/task_queues.js:80:21)
```
</details>

Reply:
So, it turns out there is a check for whether it is the `exampleCourse` to trigger that status...

https://github.com/PrairieLearn/PrairieLearn/blob/31ba617f20b1d18cdb5c15f61de0b8a6601ad258/pages/instructorFileEditor/instructorFileEditor.js#L70-L76

Reply:
To satisfy this check, there are two options:

1. Modify the help server to be scoped to the `exampleCourse` in this instance; or,
2. Treat the `testCourse` like the `exampleCourse` here:

https://github.com/PrairieLearn/PrairieLearn/blob/50f373471b24e32661295565278be1276499cd69/lib/course-db.js#L93-L97

Any preferences? @mwest1066 @trombonekenny @nwalters512

Reply:
Implemented the first option:

> Modify the helpServer to be scoped to the exampleCourse in this instance.

Diff: https://github.com/PrairieLearn/PrairieLearn/pull/1857/commits/0c80952f26c8a415a7a62adf84eed7028a364867#diff-cbe1aeb0dd1b9f30cb3f6b7f4d6bd50fR266